File: Python/Program/Vehicle.py
class Vehicle:
    def __init__(self, plat_nomor, merek, tahun_prod, warna):
        self.plat_nomor = plat_nomor
        self.merek = merek
        self.tahun_prod = tahun_prod
        self.warna = warna

    def get_plat_nomor(self):
        return self.plat_nomor

    def set_plat_nomor(self, plat_nomor):
        self.plat_nomor = plat_nomor

    def get_merek(self):
        return self.merek

    def set_merek(self, merek):
        self.merek = merek

    def get_tahun_prod(self):
        return self.tahun_prod

    def set_tahun_prod(self, tahun_prod):
        self.tahun_prod = tahun_prod

    def get_warna(self):
        return self.warna

    def set_warna(self, warna):
        self.warna = warna

File: CPP/Program/Vehicle.cpp
#pragma once

#include "Libraries.hh"

class Vehicle {
private:
    string plat_nomor;
    string merek;
    int tahun_prod;
    string warna;

public:
    Vehicle(string plat_nomor, string merek, int tahun_prod, string warna) : plat_nomor(plat_nomor), merek(merek), tahun_prod(tahun_prod), warna(warna) {}

    string get_plat_nomor() {
        return plat_nomor;
    }

    void set_plat_nomor(string plat_nomor) {
        this->plat_nomor = plat_nomor;
    }

    string get_merek() {
        return merek;
    }

    void set_merek(string merek) {
        this->merek = merek;
    }

    int get_tahun_prod() {
        return tahun_prod;
    }

    void set_tahun_prod(int tahun_prod) {
        this->tahun_prod = tahun_prod;
    }

    string get_warna() {
        return warna;
    }

    void set_warna(string warna) {
        this->warna = warna;
    }
};
